js/components/controlBar.js

```diff
@@ -577,21 +445,16 @@ var ControlBar = React.createClass({
 
     var controlItemTemplates = {
       playPause: (
-        <AccessibleButton
+        <ControlButton
+          {...commonButtonProps}
           key={CONSTANTS.CONTROL_BAR_KEYS.PLAY_PAUSE}
-          className="oo-play-pause oo-control-bar-item"
```

Comment:
Do we no longer use the class oo-control-bar-item? Some customers may have some custom CSS on this class, so we may want to document this change if so.

Reply:
`ControlButton`'s will have a `oo-control-bar-item` class by default. You can set an additional class, but they all have this as a base class.

Reply:
Got it